processing/src/main/java/io/druid/query/DruidProcessingConfig.java

```diff
@@ -31,6 +31,12 @@ public int intermediateComputeSizeBytes()
     return 1024 * 1024 * 1024;
   }
 
+  @Config({"druid.computation.buffer.pool.cacheSize", "${base_path}.buffer.pool.cacheSize"})
```

Comment:
Since this is not bytes, suggest changing name. Also, this only applies to offHeap right?